lib/dialects/postgres/query-generator.js

```diff
@@ -758,7 +753,10 @@ var QueryGenerator = {
       values = dataType.toString().match(/^ENUM\(.+\)/)[0];
     }
 
-    var sql = 'CREATE TYPE ' + enumName + ' AS ' + values + ';';
+    var schema = tableName.schema || options.schema || 'public';
+    var delimiter = tableName.delimiter || options.delimiter || '.';
+
+    var sql = 'CREATE TYPE ' + this.quoteIdentifier(schema) + delimiter + enumName + ' AS ' + values + ';';
```

Comment:
Why this change - the name creation is intentionally moved to a function, which handles schema? Isn't the end output the same?